enderio-base/src/main/java/com/enderio/api/recipe/EnchanterRecipe.java

```diff
@@ -115,7 +116,7 @@ public boolean matches(Container pContainer, Level pLevel) {
         if (!input.test(pContainer.getItem(1)) || pContainer.getItem(1).getCount() < inputAmountPerLevel) {
             return false;
         }
-        if (!pContainer.getItem(2).is(Items.LAPIS_LAZULI) || pContainer.getItem(2).getCount() < getLapisForLevel(
+        if (!pContainer.getItem(2).is(Tags.Items.GEMS_LAPIS) || pContainer.getItem(2).getCount() < getLapisForLevel(
```

Comment:
How do you mean, would you want something like enderio:enchanting_item instead?

Reply:
Well it should be handled by forge imo, so other items can also be used in the enchanter. We can implement it, but I'm not sure we should, since we don't actually add any "enchanting item". just a passing remark, nothing serious.